controllers/persist/event/events_event_handler.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Alibaba Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package event
+
+import (
+	"github.com/alibaba/kubedl/controllers/persist/util"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var _ handler.EventHandler = &enqueueForEvent{}
+
+// enqueueForEvent implements EventHandler interface and filter out events
+// not provided by kubedl managed objects.
+type enqueueForEvent struct{}
+
+func (e *enqueueForEvent) Create(evt event.CreateEvent, queue workqueue.RateLimitingInterface) {
+	evtObj := evt.Object.(*corev1.Event)
+	if !util.IsKubeDLManagedObjectKind(evtObj.InvolvedObject.Kind) {
```

Comment:
the kind of pod must be considered in this condition

Reply:
done